.github/workflows/lint-build-test.yml

```diff
@@ -1,14 +1,12 @@
 name: Lint, Build, and Test
 
 on:
-  push:
-    branches: [main]
-  pull_request:
+  workflow_call:
 
 jobs:
-  lint-build-test:
-    name: Lint, Build, and Test
-    runs-on: ubuntu-20.04
+  prepare:
+    name: Prepare
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [14.x, 16.x]
```

Comment:
Should we run this on `18.x` and `19.x` as well, to match the module template?

Reply:
I'm going to do this in a separate PR if you don't mind. I pushed up a commit that added this and it looks like the KeyringController tests are failing on 18.x: https://github.com/MetaMask/controllers/actions/runs/3446363438/jobs/5751183638#step:5:746